.travis.yml

```diff
@@ -1,10 +1,26 @@
 language: generic
 dist: trusty
-addons:
-  apt:
-    packages:
-      - emacs
+env:
+  global:
+    - CURL="curl -fsSkL --retry 9 --retry-delay 9"
+  matrix:
+    - EMACS_VERSION=24.3
+    - EMACS_VERSION=25
```

Comment:
I know I suggested version 25, but may be 25.3 is better, because otherwise you see this in the site footer:

![image](https://user-images.githubusercontent.com/3578197/33735818-6ed3bbf2-db5e-11e7-97a5-f1543c7718e7.png)